packages/liveblocks-core/src/room.ts

```diff
@@ -970,7 +970,7 @@ export function createRoom<
         authenticationSuccess(prevToken.parsed, socket);
         return undefined;
       } else {
-        return auth(config.roomId)
+        void auth(config.roomId)
```

Comment:
Chosing to _not_ await a promise should never be done carelessly/accidentally/implicitly. Here, we're deliberately ignoring the returned Promise, which we can express using the `void` keyword. This will not have a runtime presence.

The ESLint rule will force you to declare for each Promise to either `await` or `void` them, so we're explicit about how we intend to handle them.